src/assets/wise5/common/color/color.ts

```diff
@@ -0,0 +1,26 @@
+export function rgbToHex(color: string, opacity?: any): string {
```

Comment:
is opacity a number type?

Reply:
Yep, opacity can be typed to a number. I added that and also added a default value. Those changes let us remove the `parseFloat(opacity || 1)` and just use the opacity param in the r, g, b calculations.